hypothesis-python/src/hypothesis/control.py

```diff
@@ -125,7 +125,7 @@ def event(value: str) -> None:
 
 
 @deprecated_posargs
-def target(observation: float, *, label: str = "") -> None:
+def target(observation: float, *, label: str = "", allow_outside_given_test: bool = False) -> None:
```

Comment:
Here's what's wrong (in `hypothesis-python/src/hypothesis/control.py`):

```diff
-def target(observation: float, *, label: str = "", allow_outside_given_test: bool = False) -> None:
```

```diff
+def target(
+    observation: float, *, label: str = "", allow_outside_given_test: bool = False
+) -> None:
```

You can run `black hypothesis-python/src` to fix this problem. Or just fix it by hands if that's easier 👍

Reply:
Thanks for that, I was unaware of black. That's awesome :smile: